homeassistant/components/foscam/__init__.py

```diff
@@ -21,9 +25,22 @@
 
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up foscam from a config entry."""
-    await hass.config_entries.async_forward_entry_setups(entry, PLATFORMS)
 
-    hass.data.setdefault(DOMAIN, {})[entry.entry_id] = entry.data
+    session = FoscamCamera(
+        entry.data[CONF_HOST],
+        entry.data[CONF_PORT],
+        entry.data[CONF_USERNAME],
+        entry.data[CONF_PASSWORD],
+        verbose=False,
+    )
+    coordinator = FoscamCoordinator(hass, session)
+
```

Comment:
We should do `coordinator.async_config_entry_first_refresh()` here

Reply:
Ok